packages/react/src/ActionBar/ActionBar.examples.stories.tsx

```diff
@@ -19,7 +19,7 @@ import {
   ThreeBarsIcon,
 } from '@primer/octicons-react'
 import {Button, Avatar, ActionMenu, IconButton, ActionList, Textarea} from '..'
-import {Dialog} from '../DialogV1'
+import {Dialog} from '../deprecated/DialogV1'
```

Comment:
This should probably be switched to the new Dialog

Reply:
Yep, there are a few stories using the deprecated dialog that we should update!